engine/apps/metrics_exporter/helpers.py

```diff
@@ -0,0 +1,227 @@
+import random
+import typing
+
+from django.apps import apps
+from django.core.cache import cache
+from django.utils import timezone
+
+from apps.alerts.constants import STATE_ACKNOWLEDGED, STATE_FIRING, STATE_RESOLVED, STATE_SILENCED
+from apps.metrics_exporter.constants import (
+    ALERT_GROUPS_RESPONSE_TIME,
+    ALERT_GROUPS_TOTAL,
+    METRICS_CACHE_LIFETIME,
+    METRICS_CACHE_TIMER,
+    METRICS_ORGANIZATIONS_IDS,
+    METRICS_ORGANIZATIONS_IDS_CACHE_TIMEOUT,
+    METRICS_RECALCULATION_CACHE_TIMEOUT,
+    METRICS_RECALCULATION_CACHE_TIMEOUT_DISPERSE,
+    METRICS_RESPONSE_TIME_CALCULATION_PERIOD,
+    AlertGroupsResponseTimeMetricsDict,
+    AlertGroupsTotalMetricsDict,
+)
+
+
+def get_organization_ids():
+    """Try getting organizations ids from cache, otherwise get from db and save values in cache"""
+    Organization = apps.get_model("user_management", "Organization")
```

Comment:
Q: Is this here because of circular import?

Reply:
I don't think so, will change

Reply:
Double-checked, there is a circular import indeed (through alerts/alert group), keeping for now.